website/source/api/secret/nomad/index.html.md

```diff
@@ -0,0 +1,275 @@
+information about the usage and operation of the Nomad backend, please see the
+[Vault Nomad backend documentation](/docs/secrets/nomad/index.html).
+
+This documentation assumes the Nomad backend is mounted at the `/nomad` path
+in Vault. Since it is possible to mount secret backends at any location, please
+update your API calls accordingly.
+
+## Configure Access
+
+This endpoint configures the access information for Nomad. This access
+information is used so that Vault can communicate with Nomad and generate
+Nomad tokens.
+
+| Method   | Path                         | Produces               |
+| :------- | :--------------------------- | :--------------------- |
+| `POST`   | `/nomad/config/access`       | `204 (empty body)`     |
+
+### Parameters
+
+- `address` `(string: <required>)` – Specifies the address of the Nomad
+  instance, provided as `"protocol://host:port"` like `"http://127.0.0.1:4646"`.
+
+- `token` `(string: <required>)` – Specifies the Nomad Management token to use.
+
+### Sample Payload
+
+```json
+{
+  "address": "http://127.0.0.1:4646",
+  "token": "adha..."
+}
+```
+
+### Sample Request
+
+```
+$ curl \
+    --request POST \
+    --header "X-Vault-Token: ..." \
+    --data @payload.json \
+    https://vault.rocks/v1/nomad/config/access
+```
+
+## Configure Lease
+
+This endpoint configures the lease settings for generated tokens.
+
+| Method   | Path                         | Produces               |
+| :------- | :--------------------------- | :--------------------- |
+| `POST`   | `/nomad/config/lease`     | `204 (empty body)` |
+
+### Parameters
+
+- `ttl` `(int: 0)` – Specifies the lease ttl provided in seconds.
+
+- `max_ttl` `(int: 0)` – Specifies the maximum ttl provided in seconds.
+
+### Sample Payload
+
+```json
+{
+  "ttl": 1800,
+  "max_ttl": 3600
+}
+```
+
+### Sample Request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    --data @payload.json \
+    https://vault.rocks/v1/nomad/config/lease
+```
+
+## Create/Update Role
+
+This endpoint creates or updates the Nomad role definition in Vault. If the role does not exist, it will be created. If the role already exists, it will receive
+updated attributes.
+
+| Method   | Path                         | Produces               |
+| :------- | :--------------------------- | :--------------------- |
+| `POST`   | `/nomad/role/:name`         | `204 (empty body)`     |
+
+### Parameters
+
+- `name` `(string: <required>)` – Specifies the name of an existing role against
+  which to create this Nomad tokens. This is part of the request URL.
+
+- `lease` `(string: "")` – Specifies the lease for this role. This is provided
+  as a string duration with a time suffix like `"30s"` or `"1h"`. If not
+  provided, the default Vault lease is used.
+
+- `policy` `(string: "")` – Comma separated list of Nomad policies the token is going to be created against. These need to be created beforehand in Nomad.
```

Comment:
If you happen to rename `policy` to `policies`, this and other instances of `policy` should be updated.

Reply:
Fixed in 12e77fac5181e32281c97118780b0cf8638e1604